pyseir/rt/infer_rt.py

```diff
@@ -0,0 +1,617 @@
+import math
+from datetime import timedelta
+import logging
+import structlog
+
+import numpy as np
+import pandas as pd
+from scipy import stats as sps
+
+from pyseir import load_data
+from pyseir.utils import TimeseriesType, get_run_artifact_path, RunArtifact
+from pyseir.rt.constants import InferRtConstants
+from pyseir.rt import plotting, utils
+
+rt_log = structlog.get_logger(__name__)
```

Comment:
I would prefer to call this `_logger` or `logger` to keep consistent across files

Reply:
Ahh. I missed the __name__ call. I wanted to have this be a new logger that was just in infer_rt scope for comprehensibility (so without args), but now I'm not sure that's what I want. I'll have to check what context gets passed in by using that logger and if it's worth it. Either way, I'll either rename it or call a new instance.